.github/workflows/task-test.yml

```diff
@@ -133,35 +117,18 @@ jobs:
               exit 1
               ;;
           esac
-      - name: Setup specific (node20)
-        if: steps.node20.outputs.supported == 'true'
-        working-directory: setup/.install
-        run: |
-          ./install_script.sh ${{ steps.mode.outputs.mode }}
-      - name: Setup specific (node20 not supported)
-        if: steps.node20.outputs.supported == 'false'
+      - name: Setup
         working-directory: .install
         run: |
           ./install_script.sh ${{ steps.mode.outputs.mode }}
-      - name: Install LLVM for sanitizer
-        if: inputs.san == 'address'
-        working-directory: setup/.install
-        run: |
-          ./test_deps/install_llvm.sh ${{ env.LLVM_VERSION }} ${{ steps.mode.outputs.mode }}
-      - name: Full checkout (node20)
-        if: steps.node20.outputs.supported == 'true'
-        uses: actions/checkout@v4
-        with:
-          submodules: recursive
-      - name: Full checkout (node20 not supported)
-        if: steps.node20.outputs.supported == 'false'
-        run: echo "Done already earlier."
       - name: Setup tests dependencies
         run: |
           .install/test_deps/common_installations.sh ${{ steps.mode.outputs.mode }}
-      - name: Install Boost
+      - name: Install LLVM for sanitizer
```

Comment:
Can we add the logic of the `verify CC and CXX` step to the script called from this step, such that we can remove `LLVM_VERSION` completely from the flow?

Reply:
Where are we using the llvm version in the flow now?